Adds 4.8 RN bug doc text (#34386)
openshift/openshift-docs: release_notes/ocp-4-8-release-notes.adoc

* Stopping the provisioning services once the installer deploys the control plane on the bare metal nodes. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1949859[BZ#1949859] for more information.

> * Adding an Ironic health check to the bootstrap Keepalived checks. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1949859[BZ#1949859] for more information.

Reviewer: Without other data, I'd revert this change because of the guidance on styling package names.

You can now use xref:../scalability_and_performance/psap-driver-toolkit.adoc#driver-toolkit[the Driver Toolkit] as a base image for driver containers so that you can enable special software and hardware devices on Kubernetes.

[id="ocp-4-8-backup-and-restore"]
=== Backup and restore

[id="ocp-4-8-etcd-snapshot"]
==== etcd snapshot enhancement

> * A new enhancement validates the status of the etcd snapshot after backup and before restore. Previously, the backup process did not validate that the snapshot taken was complete, and likewise, the restore process did not verify that the snapshot being restored was valid, and not corrupted. Now, if the disk is corrupted during backup or restore, the error is clearly reported to the admin. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1965024[BZ#1965024].

Reviewer: ```suggestion
* A new enhancement validates the status of the etcd snapshot after backup and before restore. Previously, the backup process did not validate that the snapshot taken was complete, and the restore process did not verify that the snapshot being restored was valid, not corrupted. Now, if the disk is corrupted during backup or restore, the error is clearly reported to the admin. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1965024[BZ#1965024].
```

+
For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1948553[*BZ#1948553*]. 

> * Previously, following an etcd process, the next process would not start until the relevant ports were released. With the addition of SO_REUSEADDR to this process, the ports can be reused immediately. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1927942[BZ#1927942].

Reviewer: This bug's a duplicate, so do we need the RN?
```suggestion
* Previously, following an etcd process, the next process did not start until the relevant ports were released. With the addition of `SO_REUSEADDR` to this process, the ports can be reused immediately. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1927942[BZ#1927942].
```

Packages and functions are usually [styled](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) with backticks.

> Previously, etcd-endpoint's ConfigMap was left empty if `network.Status.ServiceNetwork` was unpopulated. As a result, the etcd Operator failed to scale up. A new feature in {product-title} 4.8 allows the etcd Operator to scale up when `network.Status.ServiceNetwork` is unpopulated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1902247[BZ#1902247])

Reviewer: ```suggestion
Previously, the etcd-endpoint's ConfigMap was left empty if the `network.Status.ServiceNetwork` field was unpopulated. As a result, the etcd Operator failed to scale up. A new feature in {product-title} 4.8 allows the etcd Operator to scale up when the `network.Status.ServiceNetwork` field is unpopulated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1902247[BZ#1902247])
```

> Generally, specify the keyword type after the keyword. Keywords are not translated; so without the keyword type, users might be confused about what the keyword represents.

*Networking*

> * Previously, an incorrect Keepalived setting could result in the VIP ending up on an incorrect system, and unable to move back to the correct system. With this update, the incorrect Keepalived setting is removed, so that the VIP ends up on the correct system. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1916890[*BZ#1916890*])

Reviewer: ```suggestion
* Previously, an incorrect Keepalived setting sometimes resulted in the VIP ending up on an incorrect system and unable to move back to the correct system. With this update, the incorrect Keepalived setting is removed so that the VIP ends up on the correct system. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1916890[*BZ#1916890*])
```
